ara/templates/host_facts.html

```diff
@@ -0,0 +1,26 @@
+{% extends "layout.html" %}
+
+{% block content %}
+<ol class="breadcrumb">
+  <li><strong>Host</strong>: {{ macros.make_link('host.show_host', host.name, host=host.name) }}</li>
+  <li class="active"><strong>Facts</strong></li>
+</ol>
+
+<h2>Last updated at {{ timestamp |datefmt }}</h2>
+<table class="facts table table-striped table-bordered table-hover table-condensed">
+  <thead>
+    <tr>
+      <th>Fact</th>
+      <th>Value</th>
+    </tr>
+  </thead>
+  <tbody>
+      {% for fact, value in facts.iteritems() %}
```

Comment:
We should be passing in `facts` as an iterator, so this should just be `for fact, value in facts`.